### ISSUE TITLE
Installation guide needs to be updated.

### ISSUE THREAD

**Issue by tommywo** [jhillyerd/inbucket#37]
For ex.
May 16, 2014 was end of life for Ubuntu 12.04

**Comment by jhillyerd** [comment]
Yup, sounds like Ubuntu is switching from upstart to systemd, so the docs will have to be reworked some.  I'm debating whether to move the install guides to the wiki to encourage user contribution/updates.

**Comment by jhillyerd** [comment]
I started on this, but will need to complete #47 first

**Comment by jhillyerd** [comment]
Due to limitations with systemd, I will need to document:

- [ ] setcap and
- [ ] /var/log/inbucket.log creation to the guide.

**Comment by jhillyerd** [comment]
Migrated inbucket.org to it's own repository: github.com/jhillyerd/inbucket.org and removed the gh-pages branch from inbucket.

**Comment by wmantly** [comment]
I see in 61c6e7c you *may* have done work to make the project set up work better with ubuntu 16.04, is there any guidance on the new set up steps for ubuntu 16.04?

**Comment by jhillyerd** [comment]
I'm planning to branch the inbucket site and work on an updated set of install instructions - last time I tried I ended up going down a rabbit hole with the Linux capabilities because my dev environment lives in a container...

I think these steps will work:

1. Follow the [outdated guide](http://www.inbucket.org/installation/ubuntu.html) through step 5.
2. Manually perform the pre-start steps (logfile creation, setcap- they only need to happen once): https://github.com/jhillyerd/inbucket/blob/master/etc/ubuntu-12/inbucket-upstart.conf#L20
3. Copy the [systemd unit file from develop](https://raw.githubusercontent.com/jhillyerd/inbucket/develop/etc/ubuntu/inbucket.service) to ` /lib/systemd/system/`
4. Install & customize inbucket.conf file (step 7 of outdated guide)
5. `systemctl enable inbucket.service`
6. `systemctl start inbucket.service`
7. `cat /var/log/inbucket.log`

Let me know if it turns out I missed anything, and I will incorporate it into the updated guide.
